src/Resources/app/administration/src/snippet/en_GB.json

```diff
@@ -2,6 +2,21 @@
     "adyen": {
         "configTestTitle": "Configuration test",
         "configTestSuccess": "Configuration is successfully tested",
-        "configTestFail": "There's an issue in your configuration"
+        "configTestFail": "There's an issue in your configuration",
+        "refundTitle": "Refund",
+        "refundSuccessful": "A refund has been successfully submitted.",
+        "invalidRefundAmount": "Refund amount exceeds order total",
+        "refundError": "An error has occurred. Please check the logs.",
+        "refundFetchError": "An error has occurred while attempting to get linked refunds.",
+        "refundConfirm": "Create refund for order",
+        "createRefund": "Create Refund",
+        "noRefundsCreated": "No refunds created",
+        "columnHeaders": {
+            "pspReference": "Adyen Reference",
```

Comment:
What do you think about setting the card title to 'Adyen Refunds' and just calling this column 'PSP Reference'?

Reply:
Updated in dbc0f94